src/controller/event/handler/init.go

```diff
@@ -31,7 +32,7 @@ func init() {
 	notifier.Subscribe(event.TopicQuotaWarning, &quota.Handler{})
 	notifier.Subscribe(event.TopicScanningFailed, &scan.Handler{})
 	notifier.Subscribe(event.TopicScanningCompleted, &scan.Handler{})
-	notifier.Subscribe(event.TopicDeleteArtifact, &scan.DelArtHandler{})
+	notifier.Subscribe(event.TopicDeleteArtifact, &scan.DelArtHandler{Context: orm.Context})
```

Comment:
Why only this handler is a little different?  Require a `Context: orm.Context`?

Reply:
This is used let's test the `DelArthandler` with mocks. 

`chart.Handler`, `p2p.Handler` and `internal.Handler` use this method to write the testcase.